src/omv/ports/webassembly/lib/oofatfs/ff.h

```diff
@@ -0,0 +1,96 @@
+/*
```

Comment:
Which files need this and why this path `lib/oofatfs/ff.h`?

Reply:
src/lib/micropython/extmod/modvfs.c#32
src/lib/micropython/extmod/vfs_fat.h#30
#include "lib/oofatfs/ff.h"

If I do not use the lib/oofatfs directory, then src/lib/micropython/lib/oofatfs/ff.h will be mixed with src/omv/ports/webassembly/oofatfs/ff.h, causing runtime errors.

This method is ugly, maybe we should write a file handle layer to include fatfs and posix

Reply:
Ah I see. It's fine for now.